Projects/Claims/Sources/Models/FlowClaimSingleItemCheckoutStepModel.swift

```diff
@@ -7,6 +7,8 @@ public struct FlowClaimSingleItemCheckoutStepModel: FlowClaimStepModel {
     let depreciation: ClaimFlowMoneyModel
     let payoutAmount: ClaimFlowMoneyModel
     let price: ClaimFlowMoneyModel
+    var payoutMethod: [ClaimAutomaticAutogiroPayoutModel] = []
```

Comment:
Not sure here if we rather want to use a struct called AvailableCheckoutMethods which can contain a ClaimAutomaticAutogiroPayoutModel?